CONTRIBUTING.md

```diff
@@ -1,64 +1,94 @@
-# Contributing to Cobalt
+# Contributing guidelines
 
-We'd love to hear about how you would like to contribute to Cobalt! It's worth
-reading through this modest document first, to understand the process and to
-make sure you know what to expect.
+## Pull Request Checklist
 
+Before sending your pull requests, make sure you do the following:
 
-## Before You Contribute
+-   Read the [contributing guidelines](CONTRIBUTING.md).
+-   Ensure you have signed the
+    [Contributor License Agreement (CLA)](https://cla.developers.google.com/).
+-   Check if your changes are consistent with the
+    [guidelines](#general-guidelines-and-philosophy-for-contribution).
```

Comment:
Suggest putting [guidelines](#general-guidelines-and-philosophy-for-contribution) and [Coding Style](#coding-style) in a lower level unordered list. So

-  Check if your changes are consistent with the:
    -  [General guidelines](#general-guidelines-and-philosophy-for-contribution)
    -  [Coding Style](#coding-style)